Update docs (#367)
NethermindEth/juno: docs/blog/2022-06-17-welcome/index.md

---

Juno is a Go implementation of a StarkNet full node client made with ❤️ by Nethermind.
## What You'll Need

- [Golang](https://go.dev/doc/install) version 1.18 for build and run the project.
- _For Linux_: You will need to install `clang`:
- Python `3.7`

```shell
sudo apt -y install clang
```

### Installing

> #### Install python

Reviewer: ```suggestion
#### Installing Python
```

> We are currently only support python `3.7`, and we recommend use pyenv. To install it, you can follow this instruction:

Reviewer: ```suggestion
Python 3.7 is the only supported version and it is recommended you use pyenv using the following instructions:
```

1. Install dependencies:

```shell
sudo apt-get update; sudo apt-get install make build-essential libssl-dev zlib1g-dev \
libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev

```
2. Install pyenv:

```shell
curl https://pyenv.run | bash
```
3. Add the following entries into your `~/.bashrc` file:

```shell
# pyenv
export PATH="$HOME/.pyenv/bin:$PATH"
eval "$(pyenv init --path)"
eval "$(pyenv virtualenv-init -)"
```
4. Restart your shell:

```shell
exec $SHELL
```
5. Install python 3.7:

```shell
pyenv install 3.7.13
pyenv global 3.7.13
```

> #### Installing golang dependencies

Reviewer: ```suggestion
#### Installing Go dependencies
```

After cloning the project,

```bash
git clone https://github.com/NethermindEth/juno
```

> You can install all the dependencies running the next command inside the project folder:

Reviewer: ```suggestion
You can install all the dependencies by running the next command inside the project root folder:
```
